### PR TITLE
PMM-5526 Fix typo in Replication dashboard description

### PR DESCRIPTION
**Pull request by eik3**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/grafana-dashboards?pullRequest=615) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/grafana-dashboards?pullRequest=615) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/grafana-dashboards?pullRequest=615) it.</sub>

**Comment by rnovikovP** [comment]
Thanks @eik3 !

@adivinho , can you  check this typo in our pmm2 dashboard as well?

File: dashboards/MySQL_Replication.json [hunk 1]
```diff
@@ -266,7 +266,7 @@
                 "rgba(36, 112, 33, 0.97)"
             ],
             "datasource": "Prometheus",
-            "description": "Displays whether the SQL Thread is in the running state or not. Applies only to a Slave host. The SQL Thread reads the events from the local relay log file and applies them to the Slave host.  Depending on the format of the binary log it can read query statements (STATEMENT format) and re-executes them, or by reading row changes (ROW format) and applying only the changes.\n\nPossibile values:\n\n**Yes** – The SQL Thread is running and is applying events from the realy log to the local Slave host\n\n**No** – The SQL Thread is not running because it is not yet started, or because of an error\n\nSee also:",
```
Comment (adivinho) [inline dashboards/MySQL_Replication.json]: Could you also replace "realy" by "relay"?

Comment (eik3) [reply dashboards/MySQL_Replication.json]: b499619 :+1:

**Pull request merged**